object_pool.go

```diff
@@ -0,0 +1,45 @@
+package astits
```

Comment:
Could you rename the file to `bytes_pool.go`?

Reply:
I planned to use this file for future implementation of *Packet and *PESData pools. If you think that it should be separated, i will do it.

Reply:
Mmm keeping them in the same file makes sense, could you just rename the file to `pools.go`?

Reply:
Done